src/chainparamsbase.h

```diff
@@ -40,10 +40,9 @@ class CBaseChainParams
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain);
 
 /**
- * Append the help messages for the chainparams options to the
- * parameter string.
+ *Set the arguments for chainparams
  */
-void AppendParamsHelpMessages(std::string& strUsage, bool debugHelp=true);
+void SetParamsArgs();
```

Comment:
For clarity I'd prefer if this was either a static class member or named appropriately, like `SetChainParamsBaseOptions`